src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/AbsoluteUriInResultGraphNode_Expected.sarif

```diff
@@ -21,12 +21,12 @@
           "ruleId": "SARIF1014",
           "level": "error",
           "message": {
+            "messageId": "Default",
             "arguments": [
-              "/runs/0/results/0/graphs/0/nodes/0/location/physicalLocation/fileLocation/uri",
+              "/runs/0/results/0/graphs/g1/nodes/0/location/physicalLocation/fileLocation/uri",
```

Comment:
>/runs/0/results/0/graphs/g1/nodes/0/location/physicalLocation/fileLocation/uri" [](start = 15, length = 79)

when will we have logical locations for these analysis target? :) is there an issue tracking it?

Reply:
Part of my test infrastructure work populates `result.fullyQualifiedLogicalName` with this JSON pointer. I assume you also want to populate the `run.logicalLocations` dictionary. Does that happen automatically if the logger detects and FQLNs?

---
In reply to: [217551974](https://github.com/Microsoft/sarif-sdk/pull/1085#discussion_r217551974) [](ancestors = 217551974)